README.md

```diff
@@ -130,8 +130,10 @@ extension has the following configuration options:
   the extension will disable the built-in VSCode JavaScript and TypeScript
   language services, and will use the Deno Language Server (`deno lsp`) instead.
   _boolean, default `false`_
-- `deno.enablePaths`: Controls if the Deno Language Server is enabled for only
+- `deno.disablePaths`: Controls if the Deno Language Server is disabled for
```

Comment:
Could you put configuration options in "h4"?

```
#### `deno.disablePaths`

Controls if the Deno...
```

Reply:
I'll do it in a later PR with the rest of the README cleanup